x-pack/plugins/integration_assistant/common/api/build_integration/build_integration.schema.yaml

```diff
@@ -0,0 +1,33 @@
+openapi: 3.0.0
```

Comment:
@maximpn Should we use `3.0.3` here?

Reply:
Technically it should be `3.0.3` but the majority of specs in Kibana use  `3.0.0`. Since it's a separate plugin it's ok to specify `3.0.3`. It should be transparent for the tooling but keep an eye on it just in case.

Reply:
Fixed in https://github.com/elastic/kibana/pull/186085/commits/ceb8d3e560a914767431617127638289d6e0a3ad